spot-ml/ml_ops.sh

```diff
@@ -46,33 +46,11 @@ else
 fi
 
 FEEDBACK_PATH=${LPATH}/${DSOURCE}_scores.csv
-DUPFACTOR=1000
-
-PREPROCESS_STEP=${DSOURCE}_pre_lda
-POSTPROCESS_STEP=${DSOURCE}_post_lda
-
-HDFS_WORDCOUNTS=${HPATH}/word_counts
-
-# paths for intermediate files
-HDFS_DOCRESULTS=${HPATH}/doc_results.csv
-LOCAL_DOCRESULTS=${LPATH}/doc_results.csv
-
-HDFS_WORDRESULTS=${HPATH}/word_results.csv
-LOCAL_WORDRESULTS=${LPATH}/word_results.csv
 
 HDFS_SCORED_CONNECTS=${HPATH}/scores
-HDFS_MODEL=${HPATH}/model
 
 LDA_OUTPUT_DIR=${DSOURCE}/${FDATE}
 
-TOPIC_COUNT=20
-
-nodes=${NODES[0]}
-for n in "${NODES[@]:1}" ; do nodes+=",${n}"; done
-
-hdfs dfs -rm -R -f ${HDFS_WORDCOUNTS}
-wait
-
 mkdir -p ${LPATH}
 rm -f ${LPATH}/*.{dat,beta,gamma,other,pkl} # protect the flow_scores.csv file
```

Comment:
At first, it can look like no one is using this folder anymore but when feedback is sent back is being sent back to that location. But yeah, deleting everything from this folder might not be necessary unless people download results from HDFS (csv) to this path as I do. 

Not sure what to do.